tests/late_joiners.py

```diff
@@ -200,8 +201,8 @@ def run(args):
 
                     # if pbft suspend the primary more than the other suspended nodes
                     if node.node_id == cur_primary_id and args.consensus == "pbft":
-                        et += et * 0.5
-                    tm = Timer(t, timeout, args=[node, True, et])
+                        et += et
```

Comment:
I am confused what this does so instead of multipling the timeout by 1.5 it is now multiplied by 2? What do either of these numbers mean? Can you please add a comment

Reply:
Sure I will add comments in all places that have raised uncertainty in this test

Reply:
For clarity, this here wasn't really been used up till now (we were never testing view changes), so this value wasn't really assessed. The reason for the increase comes from the need to make sure that view changes will be triggered, and so making the primary node stay suspended for double the view change timeout (which defaults to `5000`) should always cause a view change to be triggered.